src/components/tutorials-landing-view/certification-content-card-link/index.tsx

```diff
@@ -1,38 +1,40 @@
 import { type CertificationContentCardLinkProps } from '../types'
 import ContentCardLink from '../content-card-link'
-import consulGraphic from './img/consul.svg'
-import terraformGraphic from './img/terraform.svg'
-import vaultGraphic from './img/vault.svg'
+import networkingAutomationGraphic from './img/consul.svg'
+import infrastructionAutomationGraphic from './img/terraform.svg'
+import securityAutomationGraphic from './img/vault.svg'
 
-const PRODUCT_SLUGS_TO_BACKGROUND_IMAGES = {
-	consul: {
-		url: consulGraphic,
+const CERTIFICATION_PROGRAM_SLUGS_TO_BACKGROUND_IMAGES = {
+	'networking-automation': {
+		url: networkingAutomationGraphic,
 		lightOrDark: 'dark',
 	},
-	terraform: {
-		url: terraformGraphic,
+	'infrastructure-automation': {
+		url: infrastructionAutomationGraphic,
 		lightOrDark: 'dark',
 	},
-	vault: {
-		url: vaultGraphic,
+	'security-automation': {
+		url: securityAutomationGraphic,
 		lightOrDark: 'light',
 	},
 }
 
 const CertificationContentCardLink = ({
```

Comment:
I would consider these `CertificationContentCardLink` changes out of scope of a well-scoped `ProductSection` PR.  No need to remove these / break this up, as I know you are moving fast but just something to be mindful of!

Reply:
The addition of `ProductSection` is what spurred these changes, so in an ideal situation would you have preferred to see the `CertificationContentCardLink` changes in a PR that came before this one?